src/core_plugins/kibana/public/management/sections/indices/create_index_pattern/create_index_pattern.js

```diff
@@ -29,37 +29,28 @@ uiModules.get('apps/management')
     nameIsPattern: false,
     expandable: false,
     nameInterval: _.find(intervals, { name: 'daily' }),
-    timeField: null,
+    timeFieldOption: null,
   };
 
   // UI state.
-  this.dateFields = null;
+  this.timeFieldOptions = null;
   this.sampleCount = 5;
   this.samples = null;
   this.existing = null;
   this.nameIntervalOptions = intervals;
   this.patternErrors = [];
   this.fetchFieldsError = null;
 
-  const TIME_FILTER_FIELD_OPTIONS = {
-    NO_DATE_FIELD_DESIRED: {
-      name: $translate.instant('KIBANA-NO_DATE_FIELD_DESIRED')
-    },
-    NO_DATE_FIELDS_IN_INDICES: {
-      name: $translate.instant('KIBANA-NO_DATE_FIELDS_IN_INDICES')
-    }
-  };
-
-  const fetchFieldList = () => {
-    this.dateFields = null;
-    this.formValues.timeField = null;
-    let fetchFieldsError;
-    let dateFields;
+  const getTimeFieldOptions = () => {
+    this.timeFieldOptions = null;
```

Comment:
I get this error when I load this page:

![image](https://cloud.githubusercontent.com/assets/1238659/26434130/a9f5031e-40bb-11e7-8dd9-5d5f4352704e.png)

It looks like it's because this line sets `timeFieldOptions` to null but then it never gets set anywhere.